server/tests/db/test_01_es6.js

```diff
@@ -327,6 +338,132 @@ tests.NumberMaxSafeInteger = function() {
       'Number.isSafeInteger(Number.MAX_SAFE_INTEGER + 1);');
 };
 
+tests.NumberMinSafeInteger = function() {
+  console.assert(Number.MIN_SAFE_INTEGER - 1 === -Math.pow(2, 53),
+      'Number.MIN_SAFE_INTEGER');
```

Comment:
So, `-2**53` it isn't actually [safe according to the definition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger), and this has the consequence that if `Number.MIN_SAFE_INTEGER` was mis-defined as -2**53 then this test would still pass.

Reply:
Changed to:
`Number.MIN_SAFE_INTEGER === -(Math.pow(2, 53) - 1)`